### DISCUSSION TITLE
Have extensive documentation in the .py files

### DISCUSSION THREAD

**Discussion by sebastian-grabsch-pexon** [Viridia-Pexon/viridia#20] [Ideas]
So currently i'm working in another project with python code. This project has a huge code base which is extremely hard to properly understand, because there is a lack of documentation inside the code. 
Therefore i suggest we spent the time to describe the parts and usages with the code / module, to make it super easy to dive into our code base.

**Comment by JaegerP** [comment]
We should use a documentation framework for that, e.g. https://go.dev/blog/godoc for go (Kind of an equivalent to javadoc). For Python there is a similar framework called sphinx. And of course for REST endpoints we should use swagger.
